lib/mock-nexmo.js

```diff
@@ -54,7 +54,8 @@ function MockNexmo(log, config) {
 
         // HACK: Enable remote tests to see what was sent
         mailer.sendMail({
-          from: `sms.${senderId}@restmail.net`,
+          from: config.smtp.sender,
+          sender: config.smtp.sender,
```

Comment:
> In the hierarchy of good reasons for doing stuff, "because it's done like that elsewhere" seems pretty flimsy

Don't I know it. It was more placing faith in the reasoning of the developer that came before me.